src/util.js

```diff
@@ -352,6 +352,14 @@ function fileShouldEndWithHardline(path) {
   return true;
 }
 
+function maybeStripLeadingSlashFromUse(name) {
+  const nameWithoutLeadingSlash = name.replace(/^\\/, "");
```

Comment:
Wouldn't it be simpler to just always return the replaced string?

Reply:
@czosel `use Exception;` and `use \Exception` with namespace have difference effect

Reply:
Oh right, you're just removing leading backslashes if there's another backslash remaining :+1: